### PR TITLE
User sees loading symbol while articles load

### PR DESCRIPTION
**Pull request by maxarvid**
Link to PT: https://www.pivotaltracker.com/story/show/180024910

### User story
```
As a visitor
In order to not go elsewhere when the page is slow to load
I would like to see a loading animation
```

### Tasks
- Add test with delay to `cy.intercept` to simulate slow load
- Create boolean `loading` local state in `HomePage` component, use `useEffect` to change it
- Create ternary in `HomePage` return statement to react based on state of `loading`
- Add semanticUI `Loader` tag to display

### Screenshots
NB: Load time is intentionally delayed in Cypress for testing purposes.
lphht1nFMi

### CODE REVIEW

File: cypress/integration/userCanNavigateTheWebsite.feature.js [hunk 1]
```diff
@@ -2,16 +2,20 @@ describe("User can browser through the app", () => {
   beforeEach(() => {
     cy.intercept("GET", "**api/articles**", {
       fixture: "indexRespondsFromApi.json",
+      delay: 5000,
     });
-
     cy.visit("/");
   });
 
-  it("it is expected to display a header", () => {
```
Comment (dorianbuck) [inline cypress/integration/userCanNavigateTheWebsite.feature.js:10]: 👍🏽

File: src/App.jsx [hunk 2]
```diff
@@ -1,20 +1,28 @@
-import React, { Suspense } from "react";
+import React from "react";
 import { Switch, Route, BrowserRouter } from "react-router-dom";
 import Header from "./components/Header";
 import Footer from "./components/Footer";
 import HomePage from "./components/HomePage";
 import Category from "./components/Category";
 import IndividualArticle from "./components/IndividualArticle";
 import i18n from "./i18n";
+// import { Loader } from "semantic-ui-react";
```
Comment (dorianbuck) [inline src/App.jsx]: Are these comments?

Comment (maxarvid) [reply src/App.jsx]: They are! Removed now.

**Review by dorianbuck — APPROVED** [review]
👍🏽

**Review by lukeperrin10 — APPROVED** [review]
Good job, simple, but effective.

**Pull request merged**